user/account/src/main/res/layout/fragment_view_profile.xml

```diff
@@ -405,7 +405,6 @@
                         android:layout_height="wrap_content"
                         android:layout_marginLeft="@dimen/content_margin_large"
                         android:layout_marginRight="@dimen/content_margin_large"
-                        android:enabled="@{viewModel.preferenceRepository.loginMode.equalsIgnoreCase(@string/otp) ? false : true}"
```

Comment:
Create read only property in your viewmodel and write logic in viewmodel itself.

Reply:
it have already removed that code :)